commands/prune.go

```diff
@@ -26,11 +26,16 @@ func (cmd *Prune) Commands() []cli.Command {
 
 // Run executes the `rig prune` command
 func (cmd *Prune) Run(c *cli.Context) error {
-	cmd.out.Spin("Cleaning up unused Docker resources...")
-	/* #nosec */
-	if exitCode := util.PassthruCommand(exec.Command("docker", "system", "prune", "--all", "--volumes")); exitCode != 0 {
-		return cmd.Failure("Failure pruning Docker resources.", "COMMAND-ERROR", 13)
+
+	if util.AskYesNo("Are you sure you want to remove all unused containers, networks, images, caches, and volumes?") {
+		/* #nosec */
+		if exitCode := util.PassthruCommand(exec.Command("docker", "system", "prune", "--all", "--volumes", "--force")); exitCode != 0 {
```

Comment:
Should we start a spinner inside here in case the command might take awhile to run?

Reply:
If we kept the spinner I think it would still go above this conditional, just inside the AskYesNo block. I'm in favor of it since it can definitely take a while to run and there is no real indication that things are happening. If we leave it off then perhaps just mention in the prompt that this may take a while to execute.

Reply:
Well, the actual prune command has a lot of output as it is cleaning up things, so the spinner will likely get in the way. I'll test it out to see.

I'm thinking no spinner, but I'll see what it looks like.

Reply:
I just did a test and I didn't see any output (so if some is being generated then the spinner is gobbling it). Interestingly, I didn't see the success message until I hit "Enter" while the spinner was running so I thought things might have actually gotten stuck. Interested to see if you get the same experience (and if so then perhaps the note that it'll will take a bit is a better course).